src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,8 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeArea;
```

Comment:
Нигде. "Feel free to add some custom fields" Вот и все)

Reply:
Предположим ты написал свой класс.
В классе есть метод, который нигде не используется и он приватный.
Нужен ли нам такой метод?
У тебя приватная переменная, которую ты нигде не используешь.

Reply:
We don't need to use this variable anywhere in this task, but it's not the point. Add setter/getter for it, cause now even if you want to use it you won't be able to because of the access modifier.